hubblestack/hec/obj.py

```diff
@@ -311,9 +387,21 @@ def _send(self, *payload):
         possible_queue = False
         for server in sorted(servers, key=lambda u: u.fails):
             log.debug('trying to send %d octets to %s', len(data), server.uri)
+            if server.outage:
+                if server.outage.last_check_age < self.outage_recheck_time:
+                    log.debug('flagged as having an outage, skipping send attempt')
+                    possible_queue = True
+                    continue
+                else:
+                    log.info('flagged as having an outage -- but it is time for a recheck')
+                    server.outage.checking()
             try:
+                # Remember that we tried to send this
+                meta_data['send_attempts'] += 1
                 r = self.pool_manager.request('POST', server.uri, body=data, headers=self.headers)
                 server.fails = 0
+                if server.outage:
+                    server.outage = False
```

Comment:
The outcome of the above 2 statements is that the value of server.outage is set to False here, no matter what the initial value was. Was this the intent?